lib/kill.js

```diff
@@ -18,9 +18,14 @@ const setKillTimeout = (kill, signal, options, killResult) => {
 	}
 
 	const timeout = getForceKillAfterTimeout(options);
-	setTimeout(() => {
+	const t = setTimeout(() => {
 		kill('SIGKILL');
-	}, timeout).unref();
+	}, timeout);
+
+	// istanbul ignore else
+	if (t.unref) {
```

Comment:
@ehmicky This still needs a code comment about why it's guarded. For example:

> Guarded because there's no `.unref` when `execa` is used in the renderer process in Electron.

Reply:
Oops, I thought this was ok to merge, sorry about that.
I've added the comment in #401.